package.json

```diff
@@ -138,16 +138,19 @@
         "@types/mocha": "^9.0.0",
         "@types/node": "14.x",
         "@types/vscode": "^1.20.2",
-        "@typescript-eslint/eslint-plugin": "^4.31.1",
-        "@typescript-eslint/parser": "^4.31.1",
+        "@typescript-eslint/eslint-plugin": "^5.10.2",
+        "@typescript-eslint/parser": "^5.10.2",
         "@vscode/test-electron": "^1.6.2",
         "@vscode/test-web": "*",
-        "eslint": "^7.32.0",
+        "eslint": "^8.8.0",
+        "eslint-config-airbnb-base": "^15.0.0",
+        "eslint-plugin-import": "^2.25.4",
         "glob": "^7.1.7",
         "mocha": "^9.1.1",
         "raw-loader": "^4.0.2",
         "ts-loader": "^9.2.5",
         "typescript": "^4.4.3",
+        "vscode": "^0.9.9",
```

Comment:
is this really needed? seems to be deprecated? https://www.npmjs.com/package/vscode

Reply:
Removed this dependency and updated the lock file

Reply:
I think it got into the code base when I experimented with upcoming type-errors. The proper usage was to include "@types/vscode". I think the removal should be fine. Good catch!